adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -55,39 +116,57 @@ + (GADVersionNumber)adapterVersion {
     version.majorVersion = components[0].integerValue;
     version.minorVersion = components[1].integerValue;
     version.patchVersion = components[2].integerValue * 100 + components[3].integerValue;
-    if (components.count >= 5) {
-      version.patchVersion = version.patchVersion * 100 + components[4].integerValue;
-    }
```

Comment:
We need this. In Cocoapods if the SDK version is `7.1.9.0`, the adapter version would be `7.1.9.0.0`.

Reply:
@Mintegral-Lee Can you take a look at this comment?

Reply:
Another bump @Mintegral-Lee.

Reply:
Seems like the SDK version was updated to 7.2.1: https://cocoapods.org/pods/MintegralAdSDK, will the future versions be 3 or 4 digits?

Reply:
@Mintegral-Lee Gentle bump on the question above.

Reply:
Please ignore previous comments,the previous comment was pendding causing the comment to be delayed.

There will only be 3 digits in the future.